tests/test_renderer.py

```diff
@@ -107,6 +107,25 @@ def test_sorted_filter_vars(self):
         actual = renderers.sorted_filter(ctx, [b, c, a])
         self.assertEqual([c, b, a], actual)
 
+    def test_sorted_filter_vars(self):
```

Comment:
copy&paste error: this shadows the previous definition of `test_sorted_filter_vars ` in line 91 and leads to the drop in test coverage codecov notifies about. What about `test_sorted_filter_stack`?

Reply:
Fixed! Thanks again @HBehrens !